config/jei/itemBlacklist.cfg

```diff
@@ -49,6 +49,7 @@ advanced {
         enchantment:enchantment.damage.all.lvl4
         rftools:machine_base
         enchantment:enchantment.fishingSpeed.lvl3
+        techguns:orecluster:4
```

Comment:
don't use jei blacklist for this, we currently hide/remove items in the postInit/mod groovy files